Dfe.Academies.External.Web/Services/ConversionApplicationRetrievalService.cs

```diff
@@ -266,9 +280,9 @@ private Status CalculateFuturePupilNumbersSectionStatus(SchoolApplyingToConvert?
 	/// <returns></returns>
 	private Status CalculateLandAndBuildingsSectionStatus(SchoolApplyingToConvert? selectedSchool)
 	{
-		return selectedSchool?.LandAndBuildings.WorksPlanned.HasValue == false
-			? Status.NotStarted
-			: Status.Completed;
+		return !string.IsNullOrWhiteSpace(selectedSchool?.LandAndBuildings.OwnerExplained)
```

Comment:
why did you flip this logic back? as I flipped it as it wasn't working

Reply:
I missed this on the merge. It shouldn't make any difference because they're both mandatory fields but I can revert this if you want?

Reply:
can you please revert, just thinking I did loads of testing on this on monday